src/components/layouts/Navigation.tsx

```diff
@@ -4,16 +4,114 @@ import Link from "next/link";
 export const Navigation = () => {
   return (
     <Header>
-      <Link href="/">down to earth.</Link>
       <Navigator>
-        <Link href="/about">about</Link>
-        <Link href="/products">products</Link>
-        <Link href="/events">events</Link>
+        <Link href="/"><img src="/assets/logo.png" alt="logo" /></Link> 
+        <ul>
+          <li><Link href="/about">about</Link></li>
+          <li><Link href="/products">products</Link></li>
+          <li><Link href="/events">events</Link></li>
+        </ul> 
+        <label htmlFor="search-bar"></label>
+        <input type="text" placeholder="검색어를 입력해주세요"/> 
+        <LoggedInContainer>
+          <img src="/assets/user.svg" alt="user icon" /> 
+          <CartIconContainer>
+            <div>
+              <span>3</span>
+            </div>
+            <img src="/assets/cart.svg" alt="cart icon" />
+          </CartIconContainer> 
+        </LoggedInContainer>
+        <JoinContainer>
+          <Link href="/login">로그인</Link>
+          <Link href="/join">회원가입</Link>
+        </JoinContainer>
       </Navigator>
     </Header>
   );
 };
 
-const Header = styled.header``;
+const Header = styled.header`
+  display: flex; 
+  justify-content: space-between; 
+  align-items: center;
+  width: 100%; 
+  height: 80px;
+  box-sizing: border-box; 
+  padding: 0 100px; 
+`
 
-const Navigator = styled.nav``;
+const Navigator = styled.nav`
+  width: 100%; 
+  display: flex; 
+  justify-content: space-between;
+  align-items: center; 
```

Comment:
p2: 코드가 전체적으로 css property 순서가 통일성이 없습니다. 위에는 display가 먼저 있고 여기는 width가 먼저 있는데 지승님 올려주신 자료 참고해서 반영해주시면 좋을 것 같습니다. 
```suggestion
  display: flex; 
  justify-content: space-between;
  align-items: center; 
   width: 100%; 
```